### PR TITLE
Add confirmation dialog on account deletion

### PR DESCRIPTION
**Pull request by przybylski**
https://github.com/nextcloud/android/issues/109 

also i did a small adjustment on UI since delete and password update button gave no visual feedback on press.

### CODE REVIEW

**Comment by AndyScherzinger** [comment]
:+1:

**Comment by AndyScherzinger** [comment]
Will have to change the latest commit. I can add whatever icon we want to use. Using @android is bad because then the app loads the OS icons which depends on the OS version and we wouldn't want ICS icons for example.

What is the icon you where aiming for @przybylski?

**Comment by przybylski** [comment]
recycle bin

**Comment by AndyScherzinger** [comment]
DONE :) https://github.com/nextcloud/android/pull/116/commits/d9716c0fe10b84ccf57bb48de4d735ec46c123e4

**Comment by AndyScherzinger** [comment]
Work fine on my phone. Can you also review @tobiasKaminski?

**Comment by tobiasKaminsky** [comment]
@AndyScherzinger please use autocompletion as you type my last name very often wrong and thus I do not get a notification ;-)
Besides that please have a look at https://github.com/nextcloud/android/issues/109#issuecomment-232146608

**Comment by AndyScherzinger** [comment]
Ups,sorry. The typos are from my phone,  so no auto complete :(

**Comment by AndyScherzinger** [comment]
@tobiasKaminsky can you :+1: if this is fine with you, so the merge will be allowed by the plugin? :)

File: res/values/strings.xml [hunk 1]
```diff
@@ -109,6 +109,7 @@
     <string name="about_title">About</string>
     <string name="change_password">Change password</string>
     <string name="delete_account">Remove account</string>
+    <string name="delete_account_warning">Delete account %s? Deleting cannot be undone.</string>
```
Comment (tobiasKaminsky) [inline res/values/strings.xml]: Very minor thing: I would do a linebreak for the second sentence.

**Comment by tobiasKaminsky** [comment]
LGTM

File: res/values/strings.xml (hunk 1 above)
Comment (AndyScherzinger) [inline res/values/strings.xml]: DONE: https://github.com/nextcloud/android/pull/116/commits/63bf982a15cc82dff3cf485c7220d15f828757e6

**Pull request merged**